### PR TITLE
fix(kre-entrypoint): fix hardcoded value for compressing messages entrypoint

### PR DESCRIPTION
**Pull request by JYisus**
## Description

Compression protocols will now take the maximum permitted payload size from NATS configuration.

## Motivation and Context

Users wanted to send payloads larger than 1MB of size.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [X] I have created tests for my code changes, and the tests are passing.
- [X] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by dfinteligenz — APPROVED** [review]

File: kre-entrypoint/src/kre_entrypoint.py [hunk 1]
```diff
@@ -183,21 +192,21 @@ async def _respond_to_grpc_stream(
...
-        if len(msg) <= MESSAGE_THRESHOLD:
+        if len(msg) <= max_msg_size:
             return msg
 
         out = gzip.compress(msg, compresslevel=COMPRESS_LEVEL)
 
-        if len(out) > MESSAGE_THRESHOLD:
-            raise Exception("compressed message exceeds maximum size allowed of 1 MB.")
+        if len(out) > max_msg_size:
+            raise CompressedMessageTooLargeException("compressed message exceeds maximum size allowed.")
```
Comment (alexrodfe) [inline kre-entrypoint/src/kre_entrypoint.py:209]: Why does this excpetion have a msg when it's init also prepares another error msg?

**Review by alexrodfe — CHANGES_REQUESTED** [review]
fantastic, just a few doubts

**Review by dfinteligenz — APPROVED** [review]

**Review by alexrodfe — APPROVED** [review]

**Pull request merged**